scripts/get_message_templates.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+path="$1"
+function getMessageTemplates() {
+  grep -R "\\.withMessageTemplate" --exclude="*Test*" --exclude="*.class" "$path" | awk -F'"' '{sub(/[ \t]+$/, "", $2); sub(/[ \t]+$/, "", $3); print $2, $3, $4}' | sort -g | uniq -u | sed "s/,/=/g" > dictionary.properties
+  echo "==== Duplicated lines ===="
+  grep -R "\\.withMessageTemplate" --exclude="*Test*" --exclude="*.class" "$path" | awk -F'"' '{sub(/[ \t]+$/, "", $2); sub(/[ \t]+$/, "", $3); print $2, $3, $4}' | sort -g | uniq -d -c | sed "s/,/=/g"
+  echo "==== End ===="
```

Comment:
I haven't investigate why, but running it on powsybl-core display the following result in the console:
```
==== Duplicated lines ====
      6   
==== End ====
INFO : dictionary.properties generated successfully.
```
The `6` corresponds to 6 empty lines. They shouldn't be reported.